lms/djangoapps/courseware/tests/test_group_access.py

```diff
@@ -183,8 +184,13 @@ def check_access(self, user, block_location, is_accessible):
         """
         DRY helper.
         """
+        # Set the partition service on the item.
+        item = modulestore().get_item(block_location)
+        item.runtime._services['partitions'] = LmsPartitionService(  # pylint: disable=protected-access
```

Comment:
@nasthagiri and @cpennington I'm struggling with how best to set the partition service for tests that depend on checking group access. There seem to be a variety of ways that tests set services, and I don't understand the differences.

In this particular case, you can see I just stuck it direction on item.runtime._services. That seems pretty ugly.

I'll add some comments to other ways this is done, as I'd like to make sure I'm not introducing technical debt into the tests.